cli/bufflib/src/artifact.rs

```diff
@@ -1,24 +1,23 @@
-extern crate tar;
-extern crate ignore;
-use std::path::Path;
-use ignore::Walk;
 use std::fs::File;
+use std::io::{BufReader, Write, Read, SeekFrom, Seek};
+use flate2::{bufread::GzEncoder, Compression};
+use ignore::Walk;
 use tar::Builder;
 use tempfile::tempfile;
-use flate2::Compression;
-use std::io::BufReader;
-use flate2::bufread::GzEncoder;
-use std::io::{Write, Read, SeekFrom, Seek};
 
 pub fn create_artifact(path: &str, output_path: &str) {
```

Comment:
maybe i'm missing something, but i think the naming are inconstant, we've both package and artifact, what's the difference? maybe there's room for both, i'm just not sure what's the distinction

Reply:
maybe save_artifact_to_path?

Reply:
or compress_and_save